sdk/lib/rtl/amd64/unwind.c

```diff
@@ -687,3 +687,71 @@ RtlRaiseException(IN PEXCEPTION_RECORD ExceptionRecord)
     RtlRaiseStatus(Status);
 }
 
+VOID
+RtlCaptureNonVolatileContextPointers(
```

Comment:
Is this a name from official MS pdb symbols? If not, and if this used only internally, wouldn't it be better to use the `Rtlp` prefix instead?

Reply:
Changed.